articles/quickstart/webapp/aspnet-core/files/Program.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Program.cs
+language: csharp
+---
+
+```csharp
+builder.Services.AddControllersWithViews();
+builder.Services.AddAuth0WebAppAuthentication(options =>
+{
+    options.Domain = builder.Configuration["Auth0:Domain"];
+    options.ClientId = builder.Configuration["Auth0:ClientId"];
+});
+
+app.UseAuthentication();
```

Comment:
Can we add what builder and app are here ? Let's create the builde at the top, and the app before using it here.

Reply:
Done. Let me know if I missed anything.